### ISSUE TITLE
darkmode: change to invert color

### ISSUE THREAD

**Issue by bo-ku-ra** [terrastruct/d2#885]
darkmode is very cooooooooooooool.
but the conteners using certain colors are invisible in darkmode.

#in lightmode
x->y
y{
style.font-color:"#FFFF00";
style.stroke:"#000000"; # invisible in darkmode
style.fill:"#000000";   # invisible in darkmode
}

#in darkmode
x->y
y{
style.font-color:"#0000FF"; #invert color
style.stroke:"#FFFFFF";     #invert color
style.fill:"#FFFFFF";       #invert color
}

**Comment by bo-ku-ra** [comment]
i get it now.
d2 dark theme is not d2's dark mode!
when the os or browser is in dark mode, the d2 output svg shows me the d2 dark theme colourset.

> d2  --theme 0 --dark-theme 1 foo.d2

d2 output foo.svg shows me the light theme in light mode. (theme 0)
and
the foo.svg also shows me the light theme on darkmode. (theme 1)

oh!  i don't need inverted colours.
i don't know what to do.

**Comment by vfosnar** [comment]
yeah, maybe we can allow users to specify these custom theme colors too, for example `style.fill: neutral 3`
